### ISSUE TITLE
removeInRFC problem in non-text formats

### ISSUE THREAD

**Issue by d3e3e3** [ietf-tools/datatracker#7539]
### Describe the issue

When you do a &lt;note removeInRFC="true"&gt;, it looks like the non-indented initial word/heading "Note" is ONLY present in the .txt output but is missing in the .pdf and the .html (and also in the htmlized text). See, for example, various formats of draft-ietf-nfsv4-delstid-04

### Code of Conduct

- [X] I agree to follow the [IETF's Code of Conduct](https://github.com/ietf-tools/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by rjsparks** [comment]
@kesara, should this be an rfc2html issue rather than one for the datatracker?

**Comment by kesara** [comment]
@rjsparks yes, you are correct. I'll transfer this to [rfc2html](https://github.com/ietf-tools/rfc2html).
